src/modules/form/components/rhf/ControlledSelect.tsx

```diff
@@ -22,6 +22,7 @@ export type ControlledSelectProps = Omit<
 
 interface Props extends ControlledSelectProps {
   placeholder?: string;
```

Comment:
I was curious about this change to add a new interface `Props` that extends `ControlledSelectProps` -- what is the reason for the separation? Seems to come from Typescript https://github.com/greenriver/hmis-frontend/pull/857/files#diff-f060165369ed70f2b53e95c90ca4ef0d6aa9d8a906b7dcc926fac7b569bc7211

Reply:
Weird, I don't know why it's like that. Typescript seems perfectly happy with the below.
```
export type ControlledSelectProps = Omit<
  GenericSelectProps<PickListOption, false, false>,
  'value' | 'onChange' | 'onBlur'
> & {
  name: string;
  control?: Control; // Optional when using FormProvider
  rules?: RhfRules;
  required?: boolean;
  helperText?: ReactNode;
  placeholder?: string;
  onChange?: (option: PickListOption | null) => void;
  setValueAs?: (option: PickListOption | null) => any; // allow transform PickListOption to desired value (to support boolean)
};

// React-Hook-Form wrapper around GenericSelect.
// This component stores a string as the field value, but passes a PickListOption to the GenericSelect. (Logic that is redundant with TableFilterItemSelect, among others)
const ControlledSelect: React.FC<ControlledSelectProps> = ({
```